demo/ContainerInteractor.swift

```diff
@@ -113,6 +114,7 @@ final class ContainerInteractor {
     
     private var speedLimits: SpeedLimits?
     private var currentSpeed: Float?
+    private var currentCountry: Country?
```

Comment:
What do you think about initializing it with `.unknown`? That way we don't need optional and assert.